rg.el

```
@@ -249,13 +251,21 @@ are command line flags to use for the search."
      type-list)))
 
 
+(defun rg-get-custom-type-aliases ()
+  "Get alist of custom type aliases.
+Any lambda elements will be evaluated, and nil results will be
+filtered out."
+  (delq nil (mapcar
```

Why do you filter out `nil`? Is this expected as a benign value to be ignored or just a way to filter out obviously wrong values?

---

I am attempting to filter out empty return values from any lambdas that returned nothing.

---

Sorry if being unclear. My point was mainly if you meant to hav `nil` as part of the interface of lambdas as custom aliases. With this implementation, the user can return `nil` and the lambda will be ignored. Is that your intention?
If that's the case, it would be good if this was better documented as part of the `rg-custom-type-aliases`. 
If this is just a way to filter out an obvious error I would prefer it being signaled to the user instead of silently dropped.

Speaking of errors, it would be good to have some sanity checks here, like applying the below function to every item of the result:
```
(defun rg-is-type-alias (alias)
  (and (consp alias)
       (stringp (car alias))
       (stringp (cdr alias))))
```
Sanity checks is not something you need to implement as part of this pr. Unless you want to, of course. :)

---

I've updated the doc string to indicate nil is a valid return value should the lambda have no aliases to add.